### ISSUE TITLE
Changing permissions doesn't work

### ISSUE THREAD

**Issue by luni3359** [flathub/com.github.tchx84.Flatseal#8]
If you toggle or change anything the changes aren't saved. I suspect the flatpak doesn't have permissions to change other permissions.

**Comment by TingPing** [comment]
It has `"--filesystem=~/.local/share/flatpak/overrides:create"`

Do you see anything in that directory?

**Comment by luni3359** [comment]
The `~/.local/share/flatpak/overrides/` directory exists but it's empty.

**Comment by tchx84** [comment]
Hi @luni3359 , can you provide more information about your system ? e.g. linux distribution and version, flatpak version and flatseal version?

**Comment by luni3359** [comment]
KDE neon User Edition 5.18
Flatpak 1.6.2
Flatseal version 1.4.1

**Comment by tchx84** [comment]
I tested a close approximation of your settings:

* KDE neon User Edition 5.18 (updated today)
* Flatpak 1.6.2 (updated from Alex's ppa today)
* Flatseal 1.4.2 (updated today,please update as well)

I can't reproduce what you describe. Flatseal creates overrides correctly, and these overrides are applied properly (e.g. installed org.gnome.gedit and removed all permissions, and of course it didn't launch afterwards).

Are there any other customizations you might have done to your system, that you can think of?

A few ideas:
* How exactly are you running Flatseal?
* Can you show the output from the following command `ls -la ~/.local/share/flatpak/overrides/` ?

**Comment by luni3359** [comment]
I run Flatseal from a shortcut, but I tried running `flatpak run com.github.tchx84.Flatseal` to see the output:
```
luni3359@laptop:~$  flatpak run com.github.tchx84.Flatseal 

(com.github.tchx84.Flatseal:2): Gtk-WARNING **: 11:22:40.110: Locale not supported by C library.
        Using the fallback 'C' locale.

(com.github.tchx84.Flatseal:2): GLib-GIO-CRITICAL **: 11:22:40.248: g_dbus_proxy_new: assertion 'G_IS_DBUS_CONNECTION (connection)' failed

(com.github.tchx84.Flatseal:2): GLib-GIO-CRITICAL **: 11:22:40.249: g_dbus_proxy_new: assertion 'G_IS_DBUS_CONNECTION (connection)' failed

(com.github.tchx84.Flatseal:2): Gjs-WARNING **: 11:22:51.599: JS ERROR: GLib.FileError: Failed to create file ?/home/luni3359/.local/share/flatpak/overrides/org.DolphinEmu.dolphin-emu.003NG0?: Permission denied
_setOverrides@resource:///com/github/tchx84/Flatseal/js/model.js:293:13
_findChangesAndUpdate@resource:///com/github/tchx84/Flatseal/js/model.js:389:9
main@resource:///com/github/tchx84/Flatseal/js/main.js:31:12
run@resource:///org/gnome/gjs/modules/package.js:222:12
@/app/bin/com.github.tchx84.Flatseal:9:1


(com.github.tchx84.Flatseal:2): GLib-CRITICAL **: 11:23:03.648: Source ID 1172 was not found when attempting to remove it
```

The `ls` is empty:
```
luni3359@laptop:~$  ls -la .local/share/flatpak/overrides/
total 8
drwxr-xr-x 2 root     root     4096 Sep 14 00:46 .
drwx------ 5 luni3359 luni3359 4096 Sep 13 23:59 ..
```

**Comment by TingPing** [comment]
> drwxr-xr-x 2 root     root     4096 Sep 14 00:46 .

Broken permissions, `chown` as your user and hope nothing else has the same problem.

**Comment by tchx84** [comment]
> > drwxr-xr-x 2 root     root     4096 Sep 14 00:46 .
> 
> Broken permissions, `chown` as your user and hope nothing else has the same problem.

Yeah, that will solve the issue.

@luni3359 Did you create that directory manually by any chance?

**Comment by luni3359** [comment]
> Yeah, that will solve the issue.
> 
> @luni3359 Did you create that directory manually by any chance?

I've been t hinking about whether or not I would, but I didn't even know it existed. I will say though that I have changed permissions manually before with `flatpak`. I don't remember what I exactly ran though... but in the end I reseted everything with this [here](https://unix.stackexchange.com/questions/507248/how-to-reset-all-manually-overriden-permissions-for-a-flatpak) believe.

It works now, thank you! @TingPing @tchx84
